<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSG</class>
 <widget class="QMainWindow" name="SSG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>메뉴</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="menuBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="map">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>지도</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mission">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>미션</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>저장</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>종료</string>
     </property>
    </widget>
    <widget class="QPushButton" name="time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>시간표</string>
     </property>
    </widget>
    <widget class="QPushButton" name="out_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="chat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>691</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>631</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>50</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/free-icon-straight-up-arrow-56749.png</normaloff>../../../../Downloads/free-icon-straight-up-arrow-56749.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="downBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>50</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../../../../Downloads/free-icon-down-arrow-58927.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="goBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/free-icon-straight-right-arrow-57116.png</normaloff>../../../../Downloads/free-icon-straight-right-arrow-57116.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="leftBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/free-icon-directional-left-arrow-symbol-56911.png</normaloff>../../../../Downloads/free-icon-directional-left-arrow-symbol-56911.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="finishBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>495</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
